src/Identifier/IdentifierConverter.php

```diff
@@ -11,22 +11,21 @@
 
 declare(strict_types=1);
 
-namespace ApiPlatform\Core\Identifier\Normalizer;
+namespace ApiPlatform\Core\Identifier;
 
 use ApiPlatform\Core\Api\IdentifiersExtractorInterface;
 use ApiPlatform\Core\Exception\InvalidIdentifierException;
-use ApiPlatform\Core\Identifier\CompositeIdentifierParser;
 use ApiPlatform\Core\Metadata\Property\Factory\PropertyMetadataFactoryInterface;
 use Symfony\Component\PropertyInfo\Type;
 
 /**
- * Identifier normalizer.
+ * Identifier converter that chains identifier denormalizers.
  *
  * @author Antoine Bluchet <soyuka@gmail.com>
  */
-class ChainIdentifierDenormalizer
+final class IdentifierConverter implements IdentifierConverterInterface
 {
-    const HAS_IDENTIFIER_DENORMALIZER = 'has_identifier_denormalizer';
+    const HAS_IDENTIFIER_CONVERTER = 'has_identifier_converter';
```

Comment:
Could be moved to the interface right?

Reply:
I'd prefer not to so that we can remove it later